app/src/test/java/com/kickstarter/viewmodels/DiscoveryViewModelTest.java

```diff
@@ -347,7 +347,7 @@ public void testClearingPages() {
         .build()
     );
 
-    this.clearPages.assertValues(Arrays.asList(0, 1, 2, 3), Arrays.asList(0, 2, 3, 4));
+    this.clearPages.assertValues(Arrays.asList(0, 1, 2, 3), Arrays.asList(0, 2, 3));
```

Comment:
o_O what is this testing? or rather, why does removing 4 make this test pass? O_o

Reply:
Have no idea at all I was actually confused about this test 🧐